presto-parquet/src/main/java/com/facebook/presto/parquet/dictionary/IntegerDictionary.java

```diff
@@ -31,7 +33,8 @@ public IntegerDictionary(DictionaryPage dictionaryPage)
         super(dictionaryPage.getEncoding());
         content = new int[dictionaryPage.getDictionarySize()];
         IntegerPlainValuesReader intReader = new IntegerPlainValuesReader();
-        intReader.initFromPage(dictionaryPage.getDictionarySize(), dictionaryPage.getSlice().getBytes(), 0);
+        final ByteBufferInputStream in = ByteBufferInputStream.wrap(ByteBuffer.wrap(dictionaryPage.getSlice().getBytes()));
```

Comment:
- Please remove `final`.
- My comment about the var args call above also applies to here.

Reply:
Done.